src/main/kotlin/com/hibob/academy/filters/AuthenticationFilter.kt

```diff
@@ -0,0 +1,40 @@
+package com.hibob.academy.filters
+
+import com.hibob.academy.service.SessionService.Companion.SECRET_KEY
+import io.jsonwebtoken.Claims
+import io.jsonwebtoken.Jws
+import io.jsonwebtoken.Jwts
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.container.ContainerRequestFilter
+import jakarta.ws.rs.core.Response
+import jakarta.ws.rs.ext.Provider
+import org.springframework.stereotype.Component
+
+
+@Component
+@Provider
+class AuthenticationFilter : ContainerRequestFilter {
+
+    override fun filter(requestContext: ContainerRequestContext){
+
+        if(requestContext.uriInfo.path == "Jwt/Login") return
```

Comment:
1. best to use lower case in http paths
2. please move this to a const

Reply:
if in my path i have something like deletePet can i use camel case?